aim/aim_store.py

```diff
@@ -317,6 +318,15 @@ def resource_to_db_type(self, resource_klass):
         return self.db_model_map.get(resource_klass)
 
     def add(self, db_obj):
+        if 'object_dict' in db_obj.__dict__:
+            if isinstance(db_obj.__dict__['object_dict'], str):
```

Comment:
We should use six.text_type instead of str in this line I think.

Reply:
yes